clj-refactor.el

```diff
@@ -2614,11 +2616,13 @@ Also adds the alias prefix to all occurrences of public symbols in the namespace
                             asts-in-bad-state) "; "))))))
 
 (defun cljr--warm-ast-cache ()
+  (setq cljr-building-ast-cache? t)
   (cljr--call-middleware-async
    (cljr--create-msg "warm-ast-cache")
    (lambda (res)
      (cljr--maybe-rethrow-error res)
      (cljr--maybe-nses-in-bad-state res)
+     (setq cljr-building-ast-cache? nil)
```

Comment:
This should be the first thing, or it's going to report `t` because an error was rethrown, right?